storybook/src/pages/amsterdam/ArticlePage/ArticlePage.tsx

```diff
@@ -26,20 +20,15 @@ export const ArticlePage = ({
   spotlightLinkLabel,
 }: ArticlePageProps) => (
   <>
-    <SkipLink href="#main">Direct naar inhoud</SkipLink>
-    <Screen maxWidth="wide">
-      <AppHeader />
-      <ArticleBreadcrumb />
-      <main id="main">
-        <ArticleHeader heading={heading} imageSrc={imageSrc} />
-        <ArticleBody
-          lead={lead}
-          paragraph1={paragraph1}
-          spotlightHeading={spotlightHeading}
-          spotlightLinkLabel={spotlightLinkLabel}
-        />
-      </main>
-      <Footer>{footer}</Footer>
-    </Screen>
+    <ArticleBreadcrumb />
+    <main id="main">
+      <ArticleHeader heading={heading} imageSrc={imageSrc} />
+      <ArticleBody
+        lead={lead}
+        paragraph1={paragraph1}
```

Comment:
Why is this called `paragraph1`?

Reply:
I wanted to make some of the content editable through the Controls panel and chose the title, lead paragraph, and first regular paragraph for it.